templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -153,17 +185,27 @@ Resources:
             Ebs:
               Encrypted: true
               DeleteOnTermination: true
-              VolumeSize: {{ disk_size }}
+              VolumeSize: !Ref {{ layer }}EC2SecondaryDiskSize
               VolumeType: "gp2"
         IamInstanceProfile:
           Name:
-            Fn::ImportValue: !Sub "${Environment}${ApplicationName}{{ instance_profile_prefix }}InstanceProfileName"
+            Fn::ImportValue:
```

Comment:
leave as sub not as join.

Reply:
See other comment, that would require messy Jinja escaping.